sdc/rabbit/exceptions.py

```diff
@@ -0,0 +1,30 @@
+class BadMessageError(Exception):
+    """Raise if a message is broken in some way that means it will never
+    succesfully process.
+
+    """
+    pass
+
+
+class DecryptError(Exception):
```

Comment:
`DecryptError` is certainly an error we use, but it's not strictly related to `rabbitmq` - it's use is very specific to the SDX decryption implementation and doesn't have wider usage.

Maybe in the context of the standalone lib it should be a `QuarantinableError` or similar (as that's what it really refers to in operation, whether that was caused by a bad decrypt or other "hold it for now; it looks a bit dodgy" error)?